bundles/binding/org.openhab.binding.ddwrt/OSGI-INF/ddwrtbinding.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2010-2016 by the respective copyright holders.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.openhab.binding.ddwrt">
+   <implementation class="org.openhab.binding.ddwrt.internal.ddwrtBinding"/>
+   
+   <service>
+      <provide interface="org.osgi.service.cm.ManagedService"/>
+      <provide interface="org.osgi.service.event.EventHandler"/>
+   </service>
+   
+   <property name="event.topics" type="String" value="openhab/command/*"/>
+   <property name="service.pid" type="String" value="org.openhab.ddwrt"/>
+   
+   <reference bind="setEventPublisher" cardinality="1..1" interface="org.openhab.core.events.EventPublisher" name="EventPublisher" policy="dynamic" unbind="unsetEventPublisher"/>
+   <reference bind="addBindingProvider" cardinality="1..n" interface="org.openhab.binding.ddwrt.ddwrtBindingProvider" name="ddwrtBindingProvider" policy="dynamic" unbind="removeBindingProvider"/>
```

Comment:
Please make sure all of your class and interface names start with a capital letter.